nano/node/json_handler.cpp

```diff
@@ -2502,6 +2502,7 @@ void nano::json_handler::node_id ()
 		response_l.put ("private", node.node_id.prv.data.to_string ());
 		response_l.put ("public", node.node_id.pub.to_string ());
 		response_l.put ("as_account", node.node_id.pub.to_account ());
+		response_l.put ("as_node_id", node.node_id.pub.to_node_id ());
```

Comment:
The "peers" RPC returns this value as "node_id" and think this should too, and maybe even deprecate "as_account" in the docs?

Reply:
Seems right. Same for docs